docs/the-new-architecture/backward-compatibility-fabric-components.md

```diff
@@ -3,10 +3,428 @@ id: backward-compatibility-fabric-components
 title: Fabric Components as Native Components
 ---
 
-This section describes the required steps to ensure that a Fabric Component can be used as a Native Component.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import constants from '@site/core/TabsConstants';
+import BetaTS from './\_markdown_beta_ts_support.mdx';
 
-The section should explain:
+:::info
+The creation of a backward compatible Fabric Component requires the knowledge of how to create a Fabric Component. To recall these concepts, have a look at this [guide](pillars-fabric-components).
 
-- How to avoid installing dependencies when they are not needed
-- The usage of compilation pragmas to avoid compiling code that requires types from the codegen
-- API uniformity in JS, so that they don’t have to import different files
+Fabric Components only work when the New Architecture is properly setup. If you already have a library that you want to migrate to the New Architecture, have a look at the [migration guide](../new-architecture-intro) as well.
+:::
+
+Creating a backward compatible Fabric Component lets your users continue leverage your library, independently from the architecture they use. The creation of such a component requires a few steps:
+
+1. Configure the library so that dependencies are not installed in the old architecture.
```

Comment:
Same as for the other PR (I would omit duplicated comments so we resolve them only once)